### PR TITLE
🪲 race condition in token_mint / multisig creation

### PR DESCRIPTION
**Pull request by ryandgoulding**
Ensure that the multisig is ready to be used prior to continuing onwards.

### CODE REVIEW

File: examples/oft-solana/tasks/solana/utils.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import { Connection, PublicKey } from '@solana/web3.js'
+import { backOff } from 'exponential-backoff'
+
+export const getAccountInfo = async (connection: Connection, publicKey: PublicKey) => {
```
Comment (nazreen) [inline examples/oft-solana/tasks/solana/utils.ts:4]: I think "getAccountInfo" should be called something else, either "getAccountInfoWithBackoff" or "confirmAccountInitialized"

**Review by nazreen — CHANGES_REQUESTED** [review]

**Comment by nazreen** [comment]
this will also help solve #1078

File: examples/oft-solana/tasks/solana/utils.ts (hunk 1 above)
Comment (DanL0) [reply examples/oft-solana/tasks/solana/utils.ts:4]: I agree. Additional alternatives include: "assertAccountReadiness", "awaitAccountInfo"..

**Review by DanL0 — APPROVED** [review]
Tested on Solana Devnet. Seems working.

**Review by DanL0 — APPROVED** [review]

**Review by nazreen — APPROVED** [review]

**Comment by nazreen** [comment]
tested. works.

**Pull request merged**